Functions/Statistics/xASL_stat_PrintStats.m

```diff
@@ -362,7 +389,7 @@
         sampleMatrix = x.S.Sets1_2Sample;
```

Comment:
Are you sure you don't want to comment out the line that is currently not used? For me, it's a bit better to keep it commented until we start really using it.

But if you say that it would be too difficult to comment it now and then uncomment it later, then we can keep it as it is.

Reply:
Sorry - I resolved this and rather copied this as an answer to your comment on this topic.